### PR TITLE
Adding list of consuming queries to consumers tab

### PR DESCRIPTION
**Pull request by thomasabraham**
ARUHA-2651

### CODE REVIEW

**Comment by thomasabraham** [comment]
👍

File: client/Pages/EventTypeDetails/View.elm [hunk 1]
```diff
@@ -715,6 +724,32 @@ renderSubscription name appsInfoUrl item =
...
+consumingQueriesPanel queryStore =
+    let
+        sqlQueriesList =
+            Store.items queryStore
+
+        count =
+            List.length sqlQueriesList
+
+        countStr =
+            pluralCount count "consuming nakadi-sql queries"
```
Comment (SergKam) [inline client/Pages/EventTypeDetails/View.elm]: It produces "consuming nakadi-sql queries**s**" for multiple queries.
This function doesn't support this case "1=y, N=es", you probably need custom code with IF here.

Comment (thomasabraham) [reply client/Pages/EventTypeDetails/View.elm]: Fixed

**Comment by thomasabraham** [comment]
👍

File: client/Pages/EventTypeDetails/View.elm [hunk 2]
```diff
@@ -734,7 +734,13 @@ consumingQueriesPanel queryStore =
             List.length sqlQueriesList
 
         countStr =
-            pluralCount count "consuming nakadi-sql queries"
+            if count == 1 then
+                String.fromInt count
+                    ++ " consuming nakadi-sql query"
```
Comment (SergKam) [inline client/Pages/EventTypeDetails/View.elm]: Can be just `consuming nakadi-sql query`. It is clear from the text that it is only one.

Comment (thomasabraham) [reply client/Pages/EventTypeDetails/View.elm]: As a header `1 consuming nakadi-sql query` sounds better.
When we say `The` it kind of gives a special meaning for the query; like even if there is more queries in the future, this one will somehow be special.

**Comment by rcillo** [comment]
👍

**Pull request merged**